panel/src/panel/content.js

```diff
@@ -1,130 +1,151 @@
+import { length } from "@/helpers/object";
 import { reactive } from "vue";
 
 /**
  * @since 5.0.0
  */
 export default (panel) => {
 	return reactive({
+		/**
+		 * API endpoint to handle content changes
+		 */
+		get api() {
+			return panel.view.props.api;
+		},
+
 		/**
 		 * Returns all fields and their values that
-		 * have been changed but not yet published
+		 * have been changed but not yet saved
 		 *
 		 * @returns {Object}
 		 */
 		get changes() {
-			return panel.app.$store.getters["content/changes"]();
+			const changes = {};
+
+			for (const field in panel.view.props.content) {
+				const changed = JSON.stringify(panel.view.props.content[field]);
+				const original = JSON.stringify(panel.view.props.originals[field]);
+
+				if (changed !== original) {
+					changes[field] = panel.view.props.content[field];
+				}
+			}
```

Comment:
Wouldn't this be right too?

```suggestion
			for (const field in this.values) {
				const changed = JSON.stringify(this.values[field]);
				const original = JSON.stringify(this.originals[field]);

				if (changed !== original) {
					changes[field] = this.values[field];
				}
			}
```

Reply:
I don't fully trust the getters in combination with reactivity tbh.